### DISCUSSION TITLE
How to hide Schema section

### DISCUSSION THREAD

**Discussion by RinMinase** [DarkaOnLine/L5-Swagger#581] [General]
- L5-Swagger Version: 8.5.0 (`composer show | grep l5-swagger`)
- PHP Version (`php -v`): 8.2.5
- OS: Dockerized Alpine Linux

### Description:
How to remove the Schemas section?
(attached image, section outlined in red)
![image](https://github.com/DarkaOnLine/L5-Swagger/assets/7098304/4cd9e16e-fddc-46ee-9718-11c3cccf01a0)

I have tried adding `'default_model_expand_depth' => -1,` in the `defaults > ui > display` section of the `l5-swagger.php` config file, whidh does not work. I got the setting name [here](https://github.com/swagger-api/swagger-ui/blob/master/docs/usage/configuration.md#display) and converted the setting name to snake/underscore case similar to the other parts of the `l5-swagger` config (I also tried it as is: `defaultModelsExpandDepth`, which also does not work).

### Steps To Reproduce:
None.

**Comment by LoonerD2** [comment]
If you have published vendor blade, you can change `SwaggerUIBundle` object. Your path can be something like `your_project_folder/resources/views/vendor/l5-swagger/index.blade.php`. Try to find `SwaggerUIBundle` in <script> section and insert `defaultModelsExpandDepth: -1` like a property. 

It should be like this:
```
<script>
    window.onload = function() {
        // Build a system
        const ui = SwaggerUIBundle({
            dom_id: '#swagger-ui',
            url: "{!! $urlToDocs !!}",
            operationsSorter: {!! isset($operationsSorter) ? '"' . $operationsSorter . '"' : 'null' !!},
            configUrl: {!! isset($configUrl) ? '"' . $configUrl . '"' : 'null' !!},
            validatorUrl: {!! isset($validatorUrl) ? '"' . $validatorUrl . '"' : 'null' !!},
            oauth2RedirectUrl: "{{ route('l5-swagger.'.$documentation.'.oauth2_callback', [], $useAbsolutePath) }}",

            requestInterceptor: function(request) {
                request.headers['X-CSRF-TOKEN'] = '{{ csrf_token() }}';
                return request;
            },

            presets: [
                SwaggerUIBundle.presets.apis,
                SwaggerUIStandalonePreset
            ],

            plugins: [
                SwaggerUIBundle.plugins.DownloadUrl
            ],

            layout: "StandaloneLayout",
            docExpansion : "{!! config('l5-swagger.defaults.ui.display.doc_expansion', 'none') !!}",
            deepLinking: true,
            filter: {!! config('l5-swagger.defaults.ui.display.filter') ? 'true' : 'false' !!},
            persistAuthorization: "{!! config('l5-swagger.defaults.ui.authorization.persist_authorization') ? 'true' : 'false' !!}",
            defaultModelsExpandDepth: -1, // <- this can hide your Schemas section

        })

        window.ui = ui

        @if(in_array('oauth2', array_column(config('l5-swagger.defaults.securityDefinitions.securitySchemes'), 'type')))
        ui.initOAuth({
            usePkceWithAuthorizationCodeGrant: "{!! (bool)config('l5-swagger.defaults.ui.authorization.oauth2.use_pkce_with_authorization_code_grant') !!}"
        })
        @endif
    }
</script>
```

Use Swagger configuration documentation to learn about config properties: [Swagger configuration properties](https://github.com/swagger-api/swagger-ui/blob/master/docs/usage/configuration.md)

Hope it helps! :)
